contracts/libraries/Account.sol

```diff
@@ -362,7 +366,11 @@ library Account {
         account._updateVQuoteBalance(-(keeperFee + insuranceFundFee));
     }
 
-    /// @notice liquidates all range positions in case the account is under water
+    /// @notice liquidates token position specified by 'poolId' in case account is underwater
+    /// @notice charges a liquidation fee to the account and pays partially to the insurance fund and rest to the keeper.
```

Comment:
Multiple notice can cause issues. You can have multi line, a tab break (\t) can work.

```
/// @notice liquidates token position specified by 'poolId' in case account is underwater
///     charges a liquidation fee to the account and pays partially to the insurance fund and rest to the keeper.
```